scipy/spatial/tests/test_distance.py

```diff
@@ -63,6 +63,7 @@
                                     russellrao, seuclidean, sokalmichener,  # noqa: F401
                                     sokalsneath, sqeuclidean, yule)
 from scipy._lib._util import np_long, np_ulong
+from scipy.conftest import skip_xp_invalid_arg
```

Comment:
Note: while `pdist` is now array API compatible, adding unit tests for it would be vastly enlarge the scope of this PR, so I would rather leave it to a later PR.